framework/src/main/java/org/apache/drill/test/framework/CmdParam.java

```diff
@@ -55,6 +55,9 @@
     @Parameter(names = {"-rf", "--runFailed"}, description = "Run tests tagged as failures", required=false)
     public boolean runFailed  = false;
 
+    @Parameter(names = {"-sR", "--skipRandom"}, description = "Skip check for random failures", required=false)
```

Comment:
We need to fix the parameter names. We have too many and there is no clear convention being followed. Let's get this in for now, as is. Later work towards cleaning up.

Reply:
Sure.